gettsim/transfers/wohngeld.py

```diff
@@ -244,41 +245,38 @@ def wohngeld_eink_abzüge_m_ab_2016(
     return freib_behinderung_m + freib_kinder_m
 
 
-def wohngeld_eink_m(
+def wohngeld_eink_m_hh(
```

Comment:
I don't quite understand why we are down to the individual level now. This seems to be all using variables defined at the household level, will be the same for all members? But if I have a variable without the `_hh` suffix, I expect that I can sum it up to get the aggregate, which does not seem the right thing to do here. But maybe I am missing something.

Reply:
Christian is right, I forgot to change the docstring, I will correct that. The function should be on the household level, using the aggregated (on hh level) income after the deductions for social insurance and taxes but before the allowances (`wohngeld_eink_vor_freib_m`) and the aggregated (on hh level) allowances (`wohngeld_eink_freib_m_hh`).